### PR TITLE
fix(FixedFooterLayout): remove background applied by elevation in desktop

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1825)

### CODE REVIEW

File: src/fixed-footer-layout.tsx [hunk 1]
```diff
@@ -68,6 +68,11 @@ const FixedFooterLayout: React.FC<Props> = ({
     }, [onChangeFooterHeight, domFooterHeight]);
 
     React.useEffect(() => {
+        // There is no elevation in desktop devices, so we avoid unnecesary calculations
+        if (!isTabletOrSmaller) {
```
Comment (marcoskolodny) [inline src/fixed-footer-layout.tsx:72]: This was not strictly necessary, but we can avoid all these calculations if we are in a desktop device

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

File: src/fixed-footer-layout.tsx (hunk 1 above)
Comment (Marcosld) [reply src/fixed-footer-layout.tsx:72]: Could you add this check to `shouldDisplayElevation` function and not do the calculations for the other cases too? Just wondering.

**Review by Marcosld — COMMENTED** [review]
I can still reproduce the issue here:
https://mistica-krbchozje-tuentisre.vercel.app/playroom/preview/#?code=N4Igxg9gJgpiBcIA8BlArmMMDO2BiMMUARgIZgDWKYAToQHYAEALgJbMA2MAvADogALCB1L9GABxqsAtqRoBPAEJpmzCPW7Aky1eoAKU2Qsb66uTQAoAlI24A+RsAC+Tu0JEAmJAHoda+gYycvJ2TozediAANCDMAjDScIgAshAAbqzYzHIgTkA
Screenshot 2024-04-15 at 17 59 42

**Comment by Marcosld** [comment]
> I can still reproduce the issue here: https://mistica-krbchozje-tuentisre.vercel.app/playroom/preview/#?code=N4Igxg9gJgpiBcIA8BlArmMMDO2BiMMUARgIZgDWKYAToQHYAEALgJbMA2MAvADogALCB1L9GABxqsAtqRoBPAEJpmzCPW7Aky1eoAKU2Qsb66uTQAoAlI24A+RsAC+Tu0JEAmJAHoda+gYycvJ2TozediAANCDMAjDScIgAshAAbqzYzHIgTkA Screenshot 2024-04-15 at 17 59 42

I don't know what happened to the link :o

**Review by Marcosld — APPROVED** [review]

File: src/fixed-footer-layout.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/fixed-footer-layout.tsx]: Discussed offline. Refactored this useEffect to make it cleaner

**Pull request merged**